frontend/src/components/common/DocumentUpload.vue

```diff
@@ -154,9 +142,16 @@ export default {
       this.setErrorAlert('Sorry, an unexpected error seems to have occurred. Try uploading your files later.');
     },
     async uploadFile(env) {
+      let fileExtensionValue;
+      if(this.smallFileExtension){
+        fileExtensionValue = getFileExtensionWithDot(this.file.name);
```

Comment:
Are you able to upload pdf and png files with this? 

I thought getFileExtensionWithDot would not work for pdf's and pngs since it would return .pdf / .png, but our accepted filetypes are application/pdf and image/png.

Thinking just this.file.type would work. <- this returns application/json, image/png, image/jpeg

Reply:
Resolved. smallFileExtension is set to false when uploading a secure exchange document